cgi-bin/LJ/CleanHTML.pm

```diff
@@ -570,7 +583,15 @@ TOKEN:
                     }
                     return $text;
                 };
-                if ($cut) {
+                if ( $opts->{preserve_lj_tags_for} ) {
+                    $opencount{'lj-cut'}++;
+                    $newdata .= qq{<lj-cut};
+                    $newdata .= qq{ text="$attr->{'text'}"}
```

Comment:
Does the text attribute need to be HTML stripped or anything? I'm guess maybe, because later uses of `$newdata` are explicitly using `$etext`?

Reply:
Ack, you're right!!! Good catch. I'll add a commit.